File: Modul 3/Soal 4/PRAK204-2210817220018-Piola Evania.c
#include <stdio.h>

int main()
{
    for(int i = 0; i <= 1; i++){
    float r, t, phi, v, l, k;
    scanf("%f %f", &r, &t);
    phi = 3.142857;
    v = phi*r*r*t;
    l = 2*phi*r*(r+t);
    k = 2*phi*r;

    printf("Volume = %.2f\n", v);
    printf("Luas = %.2f\n", l);
    printf("Keliling = %.2f\n\n", k);
    }
}

File: Modul 3/Soal 4/PRAK204-2210817220018-Piola Evania.py
r = float(input(""))
t = float(input(""))
phi = 3.142857
v = phi*r*r*t
l = 2*phi*r*(r+t)
k = 2*phi*r

print("Volume = {:.2f}".format(v))
print("Luas = {:.2f}".format(l))
print("Keliling = {:.2f}\n".format(k))

r2, t2 = input().split()
r2 = float(r2)
t2 = float(t2)
phi2 = 3.142857
v2 = phi2*r2*r2*t2
l2 = 2*phi2*r2*(r2+t2)
k2 = 2*phi2*r2

print("Volume = {:.2f}".format(v2))
print("Luas = {:.2f}".format(l2))
print("Keliling = {:.2f}".format(k2))